src/sc.spatialfeature.js

```diff
@@ -16,7 +16,7 @@ export function spatialFeature(storeId, layerId, obj) {
     type: 'Feature'
   };
 
-  spatialFeature = _.defaults(defaultObj, obj);
+  spatialFeature = _.defaults(obj, defaultObj);
```

Comment:
I believe the original way was correct here.  According to [the docs](https://lodash.com/docs#defaults), the first object is the one we want to mutate.  So we want any new spatial feature object to have the properties of `defaultObj`, and if any of those properties are undefined (like identifier), then we want to use the properties of `obj` to be set on the `defaultObj`.